src/modules/tracker.js

```diff
@@ -1085,10 +1085,13 @@ class Tracker {
 
   /**
    * Send purchase event to API
-   *
    * @function trackPurchase
    * @param {object} parameters - Additional parameters to be sent with request
-   * @param {object[]} parameters.items - List of product item objects
+   * @param {{itemId: string | undefined
+   * variationId: string | undefined
+   * itemName: string | undefined
+   * count: number | undefined
+   * price: number | undefined}[]} parameters.items - List of product item objects
```

Comment:
This looks fine to me but wanted to take a second opinion @esezen

Reply:
This could be a typedef in jsdoc but I think that might be overkill since it´s not going to be used outside of here

Reply:
I think this breaks jsdoc 🤔 
![Screenshot 2024-04-02 at 4 15 57 PM](https://github.com/Constructor-io/constructorio-client-javascript/assets/2954126/c65e38df-477c-4d6b-bfc9-c6bcb2895631)

Reply:
If there is no easy way, IMO, it's okay if we only have the example and not document the type here

Reply:
This is weird...It seems to be picked up on fine by the IDE?